### ISSUE TITLE
BetterForums

### ISSUE THREAD

**Issue by void6670** [Aliucord/plugin-requests#1081]
### Discord Account

! Voidii#2870

### Plugin Description

it would be nice if the forums actually worked in the kotlin (126.21) version that Aliucord is rn, the only thing that is preventing them from working are the tags which can be implemented and any other things that might cause them not to work on this version (this could also just be a core feature)

### Request Agreement

- [X] I have made sure my plugin is possible and abides by the rules!

### ​

- [X] I did indeed check to make sure my plugin request is original!

**Comment by admin-elvistrujanovic** [comment]
Yeah, this should be added too - maybe some integration that can also exist due to the nature of plugin I requested #1104

**Comment by Palloxin** [comment]
This plugin also acts as bug fixer since on Aliucord one CAN'T create posts on Forums that __require__ the user to choose a label or tag to create the post.
This is the label (or tag) i am talking about:
![IMG_20240123_115353](https://github.com/Aliucord/plugin-requests/assets/75091899/8c2348d5-728b-47c5-b202-60fb97537893)

This is what happens if one tries to do so without a label/tag on Aliucord:
![IMG_20240121_142809](https://github.com/Aliucord/plugin-requests/assets/75091899/0eab7286-84d4-4b10-ab4a-9ce5d4ed0f20)

So it is much needed.

**Comment by tsyqax** [comment]
I have implemented a simple feature to add tags when writing a forum post.

When you press the submit button, a tag selection window appears, and you can select a tag.
Tag can only be added during the initial post creation.
(Tag editing is not implemented.)

If the forum channel does not have tags, the tag selection window will not appear.
(You must add tags to channel first to insert them into the post.)

Only one tag is visible in the forum list,
but if you go directly into the post, multiple tags are applied correctly.

Please let me know if there are any bugs.
Also, this is a somewhat temporary solution.

Note: The tag selection UI is experimental, so it may be odd.

I tested: Multiple tags, No tags, Single tag, File attachment, Text post

----

https://github.com/tsyqax/aliucord_plugins_tsq/raw/builds/ForumTagFix.zip

**Comment by Palloxin** [comment]
@tsyqax tried it, doesnt work  Could u try to make that plugin to be "published" in the #new-plugins channel of discord? Thanks!!!

**Comment by tsyqax** [comment]
> [@tsyqax](https://github.com/tsyqax) tried it, doesnt work Could u try to make that plugin to be "published" in the #new-plugins channel of discord? Thanks!!!

I cannot upload at that channel due to no permission.  
The plugin is not working? hmm  
It's working at least me, when I tried.  

Can you explain more by making issue on https://github.com/tsyqax/aliucord_plugins_tsq ?  
Then, I can review plugin easier.

**Comment by Palloxin** [comment]
Just ask mods or above, they prob will let u

**Comment by Ushie** [comment]
If someone from the community is willing to review the code in the repository to ensure there's no risk for the accounts using them then yea

**Comment by Palloxin** [comment]
Oh I just noticed that to chose tags one has to send <img width="720" height="913" alt="Image" src="https://github.com/user-attachments/assets/279f1a5a-08ca-40f3-97e2-96ed782ebf96" />

It is kinda empty here

<img width="720" height="1160" alt="Image" src="https://github.com/user-attachments/assets/0dc181e8-41d0-4a62-b3c4-090bae17e09d" />



Maybe add a lil visual clue to make one notice the plugin is ON? (second image)
